src/com/facebook/buck/ide/intellij/templates/ij-module.st

```diff
@@ -58,15 +58,21 @@
       <%generatedFolder.type% url="%generatedFolder.url%" %\\%
         isTestSource="%generatedFolder.isTestSource%" generated="true" %\\%
         %if(generatedFolder.packagePrefix)%packagePrefix="%generatedFolder.packagePrefix%" %endif%%\\%
-        %if(generatedFolder.isAndroidResources)%type="java-resource" %endif%/>
+        %if(generatedFolder.isResourceFolder)%%\\%
+          type="%generatedFolder.resourceFolderType%" %\\%
+          %if(generatedFolder.relativeOutputPath)%relativeOutputPath="%generatedFolder.relativeOutputPath%" %endif%%\\%
+        %endif%/>
     </content>
 }%%contentRoots:{contentRoot |
     <content url="%contentRoot.url%">
 %contentRoot.folders:{folder |
       <%folder.type% url="%folder.url%" %\\%
         isTestSource="%folder.isTestSource%" %\\%
         %if(folder.packagePrefix)%packagePrefix="%folder.packagePrefix%" %endif%%\\%
-        %if(folder.isAndroidResources)%type="java-resource" %endif%/>
+        %if(folder.isResourceFolder)%%\\%
+          type="%folder.resourceFolderType%" %\\%
+          %if(folder.relativeOutputPath)%relativeOutputPath="%folder.relativeOutputPath%" %endif%%\\%
```

Comment:
Being able to build and run in IntelliJ improves our iteration speed. Specifically, a lot of built-in features (e.g. automatically attaching debugger, more informative testing UI, easier-to-customize run/debug configurations) in IntelliJ are better for our workflows than command-line buck.

The IntelliJ buck plugin is an okay alternative, but for now doesn't have enough features compared to native IntelliJ for our liking (though I'm also adding a couple of changes there, hopefully we can start using it soon), so being able to support this natively in IntelliJ in addition would be really nice for us.